src/14_cal.py

```diff
@@ -21,4 +21,31 @@
 
 import sys
 import calendar
-from datetime import datetime
\ No newline at end of file
+from datetime import datetime
+
+yy = input("Input Year: ")
```

Comment:
Nice job working through this. I would check this code out as well, which may give you a slightly different result. 
```

l = len(sys.argv)

if l == 1:
    # User didn't specify any input
    month = datetime.now().month
    year = datetime.now().year
elif l == 2:
    # User didn't specify year
    month = int(sys.argv[1])
    year = datetime.now().year
elif l == 3:
    # User specified both month and year
    month = int(sys.argv[1])
    year = int(sys.argv[2])
else:
    # User provided faulty input
    print("usage: calendar.py [month] [year]")
    sys.exit(1)

cal = calendar.TextCalendar()

cal.prmonth(year, month)
```